.travis.yml

```diff
@@ -1,4 +1,5 @@
 sudo: required
+language: minimal
```

Comment:
This is really nice, I have been trying to figure out how to do this for quite some time....